rollup.config.js

```diff
@@ -1,27 +1,53 @@
-import commonjs from 'rollup-plugin-commonjs'
+import commonJs from 'rollup-plugin-commonjs'
 import nodeResolve from 'rollup-plugin-node-resolve'
 import json from 'rollup-plugin-json'
 import buble from 'rollup-plugin-buble'
+import { terser } from 'rollup-plugin-terser'
 
 const pkg = require('./package.json')
 
-export default {
-  input: './src/index.js',
-  output: {
-    file: pkg.main,
-    format: 'umd',
-    name: 'VueMeta',
-    banner: `/**
+const banner =  `/**
  * vue-meta v${pkg.version}
  * (c) ${new Date().getFullYear()} Declan de Wet & Sébastien Chopin (@Atinux)
  * @license MIT
  */
 `.replace(/ {4}/gm, '').trim()
+
+const baseConfig = {
```

Comment:
I have created separate builds for umd and cjs, so I was thinking cant we just strip all ssr stuff from the umd build now? It would make the package quite a bit smaller and you dont need ssr stuff when you use vue-meta from a cdn (but would need to have a look if thats possible at all, maybe our rollup master @pi0 has some suggestions?)

Reply:
@pimlie You are right. We can have a different entry point (browser.js) for UMD build. Also adding es build would make it double awesome.

Reply:
This has been added. Not sure if its useful, but to be sure I have added a stub for the inject method so the API is at least complete. Do you prefer I also print a warning or throw an error?